skactiveml/visualization/_feature_space.py

```diff
@@ -4,14 +4,13 @@
 
 from matplotlib import pyplot as plt
 from matplotlib.axes import Axes
-from matplotlib.colors import Colormap
 from sklearn.base import ClassifierMixin
 from sklearn.neighbors import KNeighborsRegressor
 
 from ..base import QueryStrategy
 from ..utils import check_scalar
 from ..utils._validation import check_type
-from ..utils._visualisation import mesh, check_bound, _get_boundary_args, \
+from skactiveml.visualization._auxiliary_functions import mesh, check_bound, _get_boundary_args, \
```

Comment:
relative import should be used, i.e., `..visualization._auxiliary_functions import ...`